website/content/en/status/report-2024-04-2024-06/clusteradm.adoc

```diff
@@ -0,0 +1,67 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+
+==== Cluster software refresh
+
+Except for the package builders and developer-facing ("dogfood") machines, the FreeBSD cluster mostly tracks stable/X branches.  This quarter, we started moving the stable/13 hosts to stable/14.
+
+At the time of this writing, there are 133 physical machines in the cluster, 48 run current, and 64 have been upgraded to stable/14.  The remaining machines are slated for upgrading or decommissioning in the near future.  Of the 290 jails in the cluster, 206 run stable/14.
+
+[.screen,subs="verbatim,attributes"]
+ 12.x: Regular   2, Jails   8
+ 13.x: Regular  19, Jails  68
+ 14.x: Regular  64, Jails 206
+>15.x: Regular  48, Jails   8
+Total: Regular 133, Jails 290
+Total installations: 423
+Runnning -RELEASE|{-p*}: 0
+Total geographic sites: 16
+----
```

Comment:
Do you need this line?  It seems to mess up the formatting for everything below.  See the attached screenshot (for the rendering on GitHub).

![image](https://github.com/user-attachments/assets/4c1c8014-3eb5-4500-88cd-7bd462402fec)

Reply:
Unsure.  I just copy/pasta-ed another instance of `[.screen...` 😉

Reply:
Small adjustments here: 
[clusteradm.adoc](https://people.freebsd.org/~dbaio/doc/report-2024-04-2024-06/clusteradm.adoc)   
[clusteradm.adoc.txt](https://people.freebsd.org/~dbaio/doc/report-2024-04-2024-06/clusteradm.adoc.txt)

Reply:
Thank you!  I force-pushed an update with these adjustments.